shell/artifacts/Common/Multiplexer.manifest

```diff
@@ -8,7 +8,7 @@
 
 shape HostedParticleShape
   HostedParticleShape(in ~a)
-  consume annotation
+  consume
```

Comment:
What does it mean to have no consume target?

Reply:
in a shape, it means that we don't enforce the name of a consumed slot.